src/transformers/modeling_gguf_pytorch_utils.py

```diff
@@ -93,6 +93,10 @@ def load_gguf_checkpoint(gguf_checkpoint_path, return_tensors=False):
     # to add this patch to ensure things work correctly on our side.
     if "llama" in architecture and "mistral" in model_name:
         updated_architecture = "mistral"
+    elif "t5" in architecture or "t5encoder" in architecture:
+        parsed_parameters["config"]["tie_word_embeddings"] = False
+        parsed_parameters["config"]["is_gated_act"] = True
```

Comment:
Why do we need to hardcode these ?

Reply:
I've found that almost every gguf versions of t5 models out there are actually flan-t5, and they need to set those config values when they get back to transformer's t5 class.
I didn't want to get deeper with unexpected risks, so I just hardcoded them. Therefore, feel free to fix it in clever way!

Reply:
Thanks for the explanation. Can you add a comment to explain this ? 
This is something that needs to be fixed since we don't have all the information in the gguf file. I will do that in a follow-up PR. This is something that can be easily done since the user needs to pass the original model path.

Reply:
Thanks for the explanation. Can you add a comment to explain this ? 
This is something that needs to be fixed since we don't have all the information in the gguf file. I will do that in a follow-up PR. This is something that can be easily done since the user needs to pass the original model path.

Reply:
I left a fixme comment on this!